tests/integration/targets/latest/tasks/circuit.yml

```
@@ -142,4 +153,27 @@
       - test_four['circuit']['commit_rate'] == 10000
       - test_four['circuit']['description'] == "Test circuit with updates"
       - test_four['circuit']['comments'] == "FASTER CIRCUIT"
-      - test_four['msg'] == "circuit Test Circuit One deleted"
+      - test_four['msg'] == "circuit Test Circuit One updated"
+
```

You need to run the same task again to verify idempotency.

```suggestion

- name: "PYNAUTOBOT_CIRCUIT 4.1: Verify circuit with custom_field idempotency"
  networktocode.nautobot.circuit:
    url: "{{ nautobot_url }}"
    token: "{{ nautobot_token }}"
    cid: Test Circuit One
    provider: Test Provider
    circuit_type: Test Circuit Type
    tenant: Test Tenant
    install_date: "2018-12-25"
    commit_rate: 10000
    custom_fields:
      my_text_custom_field: test
    description: "Test circuit with updates"
    comments: "FASTER CIRCUIT"
    state: present
  register: test_four

- name: "PYNAUTOBOT_CIRCUIT 5: ASSERT - Verify custom_field idempotency"
  assert:
    that:
      - not test_four['changed']
      - test_four['circuit']['custom_fields']['my_text_custom_field']  == "test"

```